src/test/java/space/obminyashka/items_exchange/service/UserServiceTest.java

```diff
@@ -80,6 +82,26 @@ void testLoginUserWithOAuth2_WhenUserBeenCreatedAndHasValidatedEmailWithOauth2Lo
         verify(userRepository, never()).setOAuth2LoginToUserByEmail(oauth2UserArgumentCaptor.capture());
     }
 
+    @Test
+    void testUpdateChildren() {
```

Comment:
You should correct the test's method name to this one: testMethodName_when..._should...